test/jdk/com/sun/tools/attach/warnings/Application.java

```diff
@@ -39,6 +45,8 @@ public static void main(String[] args) throws Exception {
 
             // send pid
             long pid = ProcessHandle.current().pid();
+            // delay 2s to allow AttachAPI initialization
+            Thread.currentThread().sleep(2000);
```

Comment:
Rather than sleep, can this determine when Attach API is initialized and ready for connections?

Reply:
Check `AttachHandler.isAttachApiInitialized()`, and delay to allow `AttachAPI` initialization if necessary.
Passed the grinder - https://openj9-jenkins.osuosl.org/job/Grinder/2957/